hash/super_tv_pc_cart.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+<!--
+license:CC0-1.0
+-->
+
+<!-- also known to exist
+Doraemon Excite Ping-Pong
+Double Mouse Party
+-->
+
+<softwarelist name="super_tv_pc_cart" description="Epoch Super TV PC cartridges">
+
+	<!-- these appear to require the system BIOS as they access above the 4MB mark for audio during the boot screen -->
+	<software name="hamatro" supported="no">
+		<description>Tottoko Hamtaro Ham Ham Ongaku Paradichu!</description>
+		<year>2004</year>
+		<publisher>Epoch</publisher>
+		<info name="alt_title" value="とっとこハム太郎 ハムハムおんがくぱらだいちゅ！"/>
+		<part name="cart" interface="super_tv_pc_cart">
+			<dataarea name="prg" size="0x400000">
+				<rom name="c-01.u1" size="0x400000" crc="0accbdcc" sha1="24fd19a87256a3335b5d1e32f11c9171f7359f32"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="doraecol" supported="no">
+		<description>Doraemon Chō Nōryoku Game Collection</description>
+		<year>2004</year>
+		<publisher>Epoch</publisher>
+		<info name="alt_title" value="ドラえもん 超能力ゲームコレクション"/>
+		<part name="cart" interface="super_tv_pc_cart">
+			<dataarea name="prg" size="0x400000">
+				<rom name="c-02.u1" size="0x400000" crc="99fb0088" sha1="b46400c850b08abd5c8ee649f483e0694b3eb059"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<!-- C-03 -->
+	
+	<software name="conan" supported="no">
+		<description>Meitantei Conan: Suiri Adventure</description>
+		<year>2005</year>
+		<publisher>Epoch</publisher>
+		<info name="alt_title" value="名探偵コナン 推理アドベンチャー"/>
```

Comment:
It should be transliterated as “Detective Conan: Suiri Adventure” as indicated on the box.

Reply:
There's several games on the software list that are titled Meitantei Conan, and at least one I can see has "Detective Conan" printed on the game title like on this box (mconant1 on psx). I just went with the same naming format used there.

Reply:
Yeah, but there are examples that use “Detective Conan” as well, e.g. in the e-kara software lists.  If there’s an official Latin script rendition of the title, we use it.

Reply:
I see! Okay, that's good to know for later reference. Those games will need a double check to see if they need renaming, but that's a thing for another day.

Reply:
fwiw mconant1 also have Detective on box/title/CD printed out, very typical Jp-to-Latin manga/anime naming export.

![s-l1600](https://github.com/mamedev/mame/assets/9251271/6e51447a-fb44-48c5-9c6b-46bbadcde371)